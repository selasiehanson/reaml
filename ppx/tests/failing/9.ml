type foo = {
  key : int;
  name : string;
}

let foo = fun [@reaml.component "Foo"] { key; name } -> Obj.magic ()
