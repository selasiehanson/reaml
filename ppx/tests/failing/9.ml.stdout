[%%ocaml.error "the `key` prop cannot be accessed inside a component"]
